data/net/minecraft/world/level/levelgen/DensityFunctions.mapping

```diff
@@ -24,6 +24,10 @@ CLASS net/minecraft/world/level/levelgen/DensityFunctions
 		ARG 0 wrapped
 	METHOD interpolated (Lnet/minecraft/world/level/levelgen/DensityFunction;)Lnet/minecraft/world/level/levelgen/DensityFunction;
 		ARG 0 wrapped
+	METHOD lerp (Lnet/minecraft/world/level/levelgen/DensityFunction;DLnet/minecraft/world/level/levelgen/DensityFunction;)Lnet/minecraft/world/level/levelgen/DensityFunction;
+		ARG 0 minFunction
+		ARG 1 value
+		ARG 3 deltaFunction
```

Comment:
```suggestion
	METHOD lerp (Lnet/minecraft/world/level/levelgen/DensityFunction;DLnet/minecraft/world/level/levelgen/DensityFunction;)Lnet/minecraft/world/level/levelgen/DensityFunction;
		ARG 0 deltaFunction
		ARG 1 min
		ARG 3 maxFunction
```

This is wrong, and so is the overload below. It roughly translates to:

```java
DF lerp(DF delta, DF min, DF max) {
    if (min is constant) {
        return lerp(delta, min, max);
    } else { 
        delta = cacheOnce(delta);
        return min * (1 - delta) + max * delta;
    }
}

DF lerp(DF delta, double min, DF max) {
    return delta * (max - min) + min;
}
```

It appears to be an optimization to avoid `cacheOnce(constant(...))`, since in order to evaluate the `lerp`, we need to evaluate either `min` or `delta` twice.

Can you **also fix the other overload** to read

```
	METHOD lerp (Lnet/minecraft/world/level/levelgen/DensityFunction;Lnet/minecraft/world/level/levelgen/DensityFunction;Lnet/minecraft/world/level/levelgen/DensityFunction;)Lnet/minecraft/world/level/levelgen/DensityFunction;
		ARG 0 deltaFunction
		ARG 1 minFunction
		ARG 2 maxFunction
```